src/ontology/phenio-odk.yaml

```diff
@@ -11,129 +12,206 @@ export_formats:
-      source: http://purl.obolibrary.org/obo/mpath.owl
-      make_base: TRUE
-      base_iris:
-        - http://purl.obolibrary.org/obo/mpath#
-        - http://purl.obolibrary.org/obo/MPATH_
-    - filename: nbo.owl
-      source: http://purl.obolibrary.org/obo/nbo.owl
-      make_base: TRUE
+    - filename: maxo.owl
+      source: http://purl.obolibrary.org/obo/maxo/maxo-base.owl
+      make_base: FALSE
       base_iris:
-        - http://purl.obolibrary.org/obo/nbo.owl#
-        - http://purl.obolibrary.org/obo/NBO_
-    - filename: ro.owl
-      source: http://purl.obolibrary.org/obo/ro.owl
-    - filename: uberon.owl
-      source: http://purl.obolibrary.org/obo/uberon.owl
-      make_base: TRUE
-      base_iris:
-        - http://purl.obolibrary.org/obo/UBERON_
-        - http://purl.obolibrary.org/obo/uberon/core#
-        - http://purl.obolibrary.org/obo/UBPROP_
+        - http://purl.obolibrary.org/obo/maxo#
+        - http://purl.obolibrary.org/obo/MAXO_
+     
+    # Phenotype Ontologies
     - filename: upheno.owl
-      source: https://github.com/obophenotype/upheno-dev/releases/latest/download/upheno.owl
-      make_base: TRUE
+      # This should be a PURL!
+      source: https://github.com/obophenotype/upheno-dev/releases/latest/download/upheno-base.owl
       base_iris:
         - http://purl.obolibrary.org/obo/UPHENO_
+    - filename: oba.owl
+      # THIS IS MISSING THE UPHENO-OBA BRIDGE
+      source: http://purl.obolibrary.org/obo/oba/oba-base.owl
+    - filename: hp.owl
+      source: http://purl.obolibrary.org/obo/hp/hp-base.owl
+      base_iris:
         - http://purl.obolibrary.org/obo/HP_
+    - filename: fypo.owl
+      source: http://purl.obolibrary.org/obo/fypo/fypo-base.owl
+      base_iris:
+        - http://purl.obolibrary.org/obo/FYPO_
+    - filename: zp.owl
+      source: http://purl.obolibrary.org/obo/zp/zp-base.owl
+      base_iris:
+        - http://purl.obolibrary.org/obo/ZP_
+    - filename: xpo.owl
+      source: http://purl.obolibrary.org/obo/xpo/xpo-base.owl
+      base_iris:
+        - http://purl.obolibrary.org/obo/XPO_
+    - filename: mp.owl
+      source: http://purl.obolibrary.org/obo/mp/mp-base.owl
+      base_iris:
         - http://purl.obolibrary.org/obo/MP_
+    - filename: wbphenotype.owl
+      source: http://purl.obolibrary.org/obo/wbphenotype/wbphenotype-base.owl
+      base_iris:
         - http://purl.obolibrary.org/obo/WBPhenotype_
-        - http://purl.obolibrary.org/obo/XPO_
-        - http://purl.obolibrary.org/obo/ZP_
-        - http://purl.obolibrary.org/obo/FYPO_
+    - filename: ddpheno.owl
+      source: http://purl.obolibrary.org/obo/ddpheno/ddpheno-base.owl
+      base_iris:
         - http://purl.obolibrary.org/obo/DDPHENO_
+    - filename: dpo.owl
+      source: http://purl.obolibrary.org/obo/dpo/dpo-base.owl
+      base_iris:
+        - http://purl.obolibrary.org/obo/FBcv_
+    
+    # Species specific anatomy Ontologies
+    # Consider replacing by Uberon metazoan?
```

Comment:
If you mean “composite-metazoan”, keep in mind what I told you about the “unfolding” that happens in that product. It may _not_ be what you need, and so that comment may be misleading for any future reader.

Reply:
I have fixed that now, thaks!